### PR TITLE
chore: move jobrunner to ~jobrunner/jobrunner

### PR DESCRIPTION
**Pull request by madwort**
* fix https://github.com/opensafely-core/backend-server/issues/57

### CODE REVIEW

**Review by bloodearnest — COMMENTED** [review]
This is much cleaner than I thought it would be nice.

One suggestion - could we put the code/lib dirs /home/jobrunner/jobrunner?

Then we have a dir for each service `jobrunner` and `release-hatch`. It looks a bit weird, but if we want to rename the jobrunner user to `opensafely`, it makes a bit more sense?

However, I think the environ dir is correct to be in ~jobrunner directly. Reading this has reminded me that the config in `environ` is actually not job-runner specific - its also used for release-hatch config, as they share a lot of things.

Thoughts?

**Comment by madwort** [comment]
> could we put the code/lib dirs in /home/jobrunner/jobrunner, e.g. /home/jobrunner/jobrunner/code ? 

yeah good shout, I'll do that now.

> environ dir is correct to be in ~jobrunner directly. Reading this has reminded me that the config in environ is actually not job-runner specific - its also used for release-hatch config, as they share a lot of things.

Yep, as discussed offline, let's move this to ~jobrunner/config & move the install for it out of `services/jobrunner/install.sh` to somewhere more general (but not in this PR)

File: services/jobrunner/install.sh [hunk 1]
```diff
@@ -54,34 +40,66 @@ EOF
     cat "$src" >> "$dst"
 }
 
-cp $SRC_DIR/bin/* $TARGET_DIR/bin/
-cp $SRC_DIR/sbin/* /usr/local/sbin
+# set up config
+
+mkdir -p $HOME_DIR
```
Comment (madwort) [inline services/jobrunner/install.sh:45]: this is unnecessary

Comment (madwort) [reply services/jobrunner/install.sh:45]:
```suggestion
```

File: services/jobrunner/bashrc [hunk 2]
```diff
@@ -19,11 +19,11 @@ Current status:
 systemctl status -n0 jobrunner
 $(systemctl status -n0 jobrunner)
 
-Environment variables have been set from /srv/jobrunner/environ/*.env"
+Environment variables have been set from /home/jobrunner/environ/*.env"
```
Comment (bloodearnest) [inline services/jobrunner/bashrc:22]:
```suggestion
Environment variables have been set from ./environ/*.env"
```

File: services/jobrunner/bashrc [hunk 3]
```diff
@@ -19,11 +19,11 @@ Current status:
...
 
-Code: /srv/jobrunner/code
-Deps: /srv/jobrunner/lib
-Config: /srv/jobrunner/environ
+Code: /home/jobrunner/jobrunner/code
```
Comment (bloodearnest) [inline services/jobrunner/bashrc:24]:
```suggestion
Code: ./jobrunner/code
```

File: services/jobrunner/bashrc [hunk 4]
```diff
@@ -19,11 +19,11 @@ Current status:
...
+Deps: /home/jobrunner/jobrunner/lib
```
Comment (bloodearnest) [inline services/jobrunner/bashrc:25]:
```suggestion
Deps: ./jobrunner/lib
```

File: services/jobrunner/bashrc [hunk 5]
```diff
@@ -19,11 +19,11 @@ Current status:
...
+Config: /home/jobrunner/environ
```
Comment (bloodearnest) [inline services/jobrunner/bashrc:26]:
```suggestion
Config: ./environ
```

**Review by bloodearnest — APPROVED** [review]
LGTM

Slight wording change suggestions in the bashrc

**Pull request merged**